practice_05.11.19_classes/Task2_Plants/Gallery.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace practice_05._11._19_classes.Task2_Plants
+{
+    public class Gallery
+    {
+        private Plant[] plants;
```

Comment:
plants and worker should be make readonly since they are not modified after first initialization in constructor.

Reply:
fixed